packages/devtools/lib/src/memory/memory.dart

```diff
@@ -680,11 +681,12 @@ class MemoryScreen extends Screen with SetStateMixin {
       ));
 
       table.setRows(instanceRows);
-    } catch (e) {
+    } catch (e, st) {
```

Comment:
nit: what is st? perhaps give it a clearer name?

Reply:
`st` is the stacktrace for the exception. I think it's pretty common that we use `st` for that, at least in my experience.

Reply:
Ah - that makes sense. Leaving it as is sgtm.